_posts/2024-05-23-pyos-pyconus-2024-recap.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: single
+title: "pyOpenSci at PyCon US 2024 - Python packaging and community"
+excerpt: "Learn about  pyOpenSci's experience at PyCon US 2024, how pyOpenSci is making Python packaging more accessible and beginner friendly and how the Python community is helping us get there."
+author: "Leah Wasser"
+permalink: /blog/recap-pyos-pyconus-2024.html
```

Comment:
better url?

Reply:
ok great. i've read so many different things about url's and SEO for a while it was like long ones with lots of words and then it was shorter and concise. and now with AI... i have no idea.  😆